spec/cypress/end_to_end/research_plan_with_user.cy.js

```diff
@@ -13,4 +13,36 @@ describe('Research Plan with User', () => {
     cy.contains('My Research Plan 2');
     cy.get('#tabList-tab-4 > span').contains('1(0)');
   });
+
+  it('rename research plan', () => {
```

Comment:
I ran this tests locally with `yarn cypress run --project spec --spec "spec/cypress/end_to_end/research_plan_with_user.cy.js"`, and it fails with

```
1) Research Plan with User
       rename research plan:
     AssertionError: Timed out retrying after 5000ms: Expected to find element: `[width="280px"] > :nth-child(1) > .preview-table`, but never found it.
      at Context.eval (webpack:///./cypress/end_to_end/research_plan_with_user.cy.js:9:7)

  2) Research Plan with User
       rename research plan:
     AssertionError: Timed out retrying after 5000ms: Expected to find element: `[width="280px"] > :nth-child(1) > .preview-table`, but never found it.
      at Context.eval (webpack:///./cypress/end_to_end/research_plan_with_user.cy.js:24:7)
```

Not sure if it fails because of a timeout on my machine or because the selector (`'[width="280px"] > :nth-child(1) > .preview-table'`) is too brittle?

Reply:
Dont know, it runs fine on my end. But anyway I added cypress identifier for this part and I hope now it should work seamlessly.

Reply:
I increased the timeout from 5000 to 10000 and tried running on chrome instead of electron, Still won't work 🤔 

```
1) Research Plan with User
       rename research plan:
     AssertionError: Timed out retrying after 10000ms: Expected to find element: `[data-cy="researchPLanItem-1"]`, but never found it.
      at Context.eval (webpack:///./cypress/end_to_end/research_plan_with_user.cy.js:9:7)

  2) Research Plan with User
       add/remove fields in research plan:
     AssertionError: Timed out retrying after 10000ms: Expected to find element: `[data-cy="researchPLanItem-1"]`, but never found it.
      at Context.eval (webpack:///./cypress/end_to_end/research_plan_with_user.cy.js:24:7)
```